app/src/main/java/com/sjh/autosummary/core/common/const/GptConst.kt

```diff
@@ -0,0 +1,14 @@
+package com.sjh.autosummary.core.common.const
+
+import com.sjh.autosummary.core.model.ChatRoleType
+import com.sjh.autosummary.core.model.MessageContent
+
+object GptConst {
+    val DEFAULT_GPT_MODEL = "gpt-4o-mini-2024-07-18"
```

Comment:
```suggestion
    const val DEFAULT_GPT_MODEL = "gpt-4o-mini-2024-07-18"
```
이전에 멘토링 시간에 언급을 했듯이 const val과 val은 다릅니다.
그 때 숙제를 하셨던가요?

Reply:
자세히는 기억이 안나지만 값이 할당되는 타이밍이 다르다고만 알고 있습니다. 이 부분에 대해서 다시 공부하겠습니다!

Reply:
상수와 read-only 변수의 차이라고 말씀 드렸었는데 기억이 안나시나보네요.

Reply:
앗... 복습을 자주 하겠습니다....